lib/public/css/docs-detail.scss

```diff
@@ -5,11 +5,14 @@
   .wrapper {
 
     .markdown-body {
-      margin-bottom: 4rem;
-      margin-left: 300px;
-      margin-top: 80px;
-
-      @media #{$small-up} {
+      margin-bottom: calc(1rem + 60px);
+      margin-left: 0;
+      margin-top: calc(2rem + 80px);
```

Comment:
Does the 80px part of the margin have to be on the markdown body?  Seems like it would ideally go higher up on the `.docs-detail` element.